identity/handler.go

```diff
@@ -220,7 +220,8 @@ func (h *Handler) create(w http.ResponseWriter, r *http.Request, _ httprouter.Pa
 		return
 	}
 
-	i := &Identity{SchemaID: cr.SchemaID, Traits: []byte(cr.Traits), State: StateActive, StateChangedAt: sqlxx.NullTime(time.Now())}
+	var time = sqlxx.NullTime(time.Now())
```

Comment:
In go it is considered bad practice to have variable names overshadow package names, so it’s best to rename this:

```suggestion
	stateChangedAt := sqlxx.NullTime(time.Now())
```